.github/workflows/import_count.yml

```
@@ -0,0 +1,57 @@
+name: Compute transitive imports
+
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.8
+
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        sudo apt-get install -y jq
+        # If you have additional dependencies, install them here
+
+    - name: Compute transitive imports
+      run: |
+        # Compute the counts for the HEAD of the PR
+        python ./scripts/count-trans-deps.py > head.json
```

```suggestion
        python ./scripts/count-trans-deps.py <missing dir argument?> > head.json
```
Should there be a dir here, otherwise the python code seems to give an error.

---

Aah, last minute update that I forgot to fix.
```suggestion
        python ./scripts/count-trans-deps.py "Mathlib/" > head.json
```